typescript/cli/src/tests/commands/helpers.ts

```diff
@@ -531,7 +531,10 @@ export async function restoreSnapshot(
   const result = await snapshotBaseCall<boolean>(rpcUrl, 'evm_revert', [
     snapshotId,
   ]);
-  assert(result, 'Failed to restore snapshot');
+
+  if (!result) {
+    throw new Error('Failed to restore snapshot');
+  }
```

Comment:
what was the issue with the assert here?

Reply:
Unnecessary to import something as simple as that

Reply:
but the assertion is like the standard across the whole codebase, I would keep it and even enforce it in the other parts